contents/blog/how-we-built-email-onboarding.md

```diff
@@ -8,54 +8,47 @@ featuredImage: ../images/blog/blog_onboarding.jpg
 featuredImageType: full
 ---
 
-At PostHog there's a lot of room for people to build things they have a personal interest in, even if it's outside their normal purview. That’s how I ended up owning PostHog’s email onboarding: I thought it was interesting and impactful, so I got involved. 
+Email is a powerful tool for getting users to engage with your product, but it can be really difficult to know how the best companies structure their onboarding email flows. You end up having to sign up to a bunch of them, then waiting and letting your inbox get flooded. 
 
-Now, with our email onboarding flow in its fifth iteration, I want to explain all the different versions we’ve been through and reflect on what we’ve learned.
+Often, you end up getting awful sales rep outreach emails from them _months_ later too. 
 
-## Onboarding 1.0 to Onboarding 1.5
-**RFC:** [Emails, Newsletters and Email Lists](https://github.com/PostHog/posthog.com/issues/3202)
-**RFC:** [Onboarding 1.5 emails](https://github.com/PostHog/posthog/issues/9784)
+We don’t do those sort of outreach emails. This is what we do instead, and how the email onboarding has changed over the last five iterations. We think it performs pretty well and defies the idea that developers don't like email marketing - but we've included performance data so you can judge for yourself. 
 
-Back in 2022 we used Mailchimp to power email announcements and a monthly changelog email. We didn’t do much email onboarding for new users outside of transactional emails, and our changelog email was opt-in only, with just 660 people signed up. Internally nobody really knew what the email experience looked like, so I mapped it in Miro.
+## Onboarding 1.0: The one where it started costing lots
+
+Back in 2022 our onboarding flow was powered by Mailchimp and was very, _very_ simple. In fact, nobody actually knew what it looked like. So, I mapped it in [an RFC](https://github.com/PostHog/posthog.com/issues/3202).
 
 ![Onboarding 1.0](../images/blog/onboarding/onboarding_1.png)
 <caption>The first map we did of what our email onboarding flow looked like</caption>
 
-Turns out we only had basic re-engagement emails which were sent if we detected a user hadn’t ingested any data – nothing else, not even a welcome email.
+We didn’t do welcome emails for new users, and even our monthly changelog emails were only sent to 660 opted-in users. I decided to change this, imported 30,000+ users into that email list, and sent our Mailchimp costs skyrocketing. Whoops.
```

Comment:
This makes it slightly sound like you opted in 30k people against their will, when I don't think that's what actually happened?

Reply:
...That is basically what happened. 

We reviewed our T&Cs to check we could, then we did. We were only using it for product update news so we figured nobody would complain. Nobody did.

Reply:
Ah ok, gotcha - yeah what you've said here even would be worth including in some form

> We reviewed our T&Cs to check we could, then we did. We were only using it for product update news so we figured nobody would complain. Nobody did.